elementary/cli/upgrade.py

```diff
@@ -3,23 +3,27 @@
 import click
 from packaging import version
 
+from elementary.config.config import Config
 from elementary.utils import package
 
 
 def recommend_version_upgrade():
-    try:
-        latest_version = package.get_latest_package_version()
-        current_version = package.get_package_version()
+    config = Config()
 
-        if not latest_version:
-            # Failed to obtain the latest version, so skip the check
-            return
+    if not config.disable_elementary_version_check:
```

Comment:
Maybe change this so that there wouldn't be a need to indent the rest of the code:

```py
    if config.disable_elementary_version_check:
        return
```

Reply:
thanks @elongl. Sorry for late reply, I've been on holiday.

This is done.